compiler/rustc_target/src/spec/armv6k_nintendo_3ds.rs

```diff
@@ -0,0 +1,45 @@
+use crate::spec::{LinkArgs, LinkerFlavor, PanicStrategy, RelocModel, Target, TargetOptions};
+
+/// A base target for Nintendo 3DS devices using the devkitARM toolchain.
+///
+/// Requires the devkitARM toolchain for 3DS targets on the host system.
+
+pub fn target() -> Target {
+    let mut pre_link_args = LinkArgs::new();
+    pre_link_args.insert(
+        LinkerFlavor::Gcc,
+        vec![
+            "-specs=3dsx.specs".to_string(),
```

Comment:
Who provides this file? devkitARM?

Reply:
Yes, together with the linker and system libraries (the file itself includes a link script and the C Runtime objects).